src/components/organize/tasks/forms/CreateTaskForm.tsx

```diff
@@ -0,0 +1,131 @@
+// import dayjs from 'dayjs';
+import { Form } from 'react-final-form';
+import { TextField } from 'mui-rff';
+import { useQuery } from 'react-query';
+import { useRouter } from 'next/router';
+import { Box, Button, MenuItem } from '@material-ui/core';
+import { FormattedMessage, FormattedMessage as Msg, useIntl } from 'react-intl';
+
+import getCampaigns from '../../../../fetching/getCampaigns';
+import { ZetkinTaskReqBody, ZetkinTaskType } from '../../../../types/zetkin';
+
+interface CreateTaskFormProps {
+    onSubmit: (data: ZetkinTaskReqBody) => void;
+    onCancel: () => void;
+}
+
+const CreateTaskForm = ({ onSubmit, onCancel }: CreateTaskFormProps): JSX.Element => {
+    const router = useRouter();
+    const { campId, orgId } = router.query as {campId: string; orgId: string};
+    const intl = useIntl();
+    const { data: campaigns } = useQuery(['campaigns', orgId], getCampaigns(orgId));
+
+    // const formattedNow = dayjs().format('YYYY-MM-DDThh:mm');
```

Comment:
Should this be removed or left for later?

Reply:
Yep you're right